pyiode/pyiode_cdef.pyx

```diff
@@ -64,7 +64,7 @@ cdef pyfloats(double *cvar, int lg):
     return res    
 
 
-cdef iodevar_to_ndarray(char * name, int copy = True):
+cdef iodevar_to_ndarray(char * name, int copy = 1):
```

Comment:
please change here to 
```C
cdef iodevar_to_ndarray(char * name, bint copy = True):
```

Please use the Cython boolean type `bint` when the cdef function expects a boolean argument, not int

Reply:
Ok.